.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!-- The following is intended to be helpful to you. Feel free to remove anything that is not. -->
+
+# JIRA Ticket: 
+MNTOR-
+
+
+<!-- When adding a new feature: -->
+
+# Description
+
+
+
+# Screenshot (if applicable)
+
+Not applicable.
+
+# How to test
+
+
+
+# Checklist (Definition of Done)
+- [ ] l10n changes have been submitted to the l10n repository, if any.
+- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
+- [ ] I've added or updated relevant docs in the docs/ directory.
```

Comment:
We don't have a docs directory yet in /src – should we add this, or try to reuse the one in root?

Reply:
Let me word it a little differently.. How about: 
`I've added or updated the relevant sections in readme and/or code comments`

Reply:
```suggestion
- [ ] I've added or updated the relevant sections in readme and/or code comments
```